examples/observability-to-the-rescue/tracetest/tracetest-provision.yaml

```diff
@@ -0,0 +1,28 @@
+---
+type: PollingProfile
+spec:
+  name: Default
+  strategy: periodic
+  default: true
+  periodic:
+    retryDelay: 5s
+    timeout: 10m
+
+---
+type: DataStore
+spec:
+  name: Jaeger
+  type: jaeger
+  default: true
+  jaeger:
+    endpoint: jaeger:16685
+    tls:
+      insecure: true
+
+# ---
+# type: Analyzer
+# spec:
+#   id: current
+#   name: Analyzer
+#   enabled: false
+#   minimumScore: 0
```

Comment:
Is there a reason for having this?

Reply:
It is because the Analyzer provisioning is broken for empty plugins. @xoscar solved it yesterday and as soon as we release a new version, I will uncomment this.